README.md

```diff
@@ -6,12 +6,16 @@
 
 Extension for using the [Lua programming language](https://www.lua.org/) in Godot 4.3+
 
+With this addon, you can create scripts directly in Lua, using it as an alternative to GDScript or C#.
+You can also create sandboxed Lua states for external modding/scripting support, as many as necessary.
+
 This plugin is available in the Asset Library as [Lua GDExtension](https://godotengine.org/asset-library/asset/2330).
 
 
 ## Features
-- Run Lua scripts, access Lua tables, functions and coroutines directly from GDScript, C# or any other scripting language in Godot
-- Create as many Lua states as you need
+- Create Godot scripts directly in Lua, making it possible to use Lua as an alternative to GDScript or C#
```

Comment:
You repeat yourself here, the same words can be found a couple of lines prior. 

I suggest

Use Lua as scripting language to program your game or application

Reply:
You're right, thanks for the review! I was a little lazy there, gotta admit 😅

> Use Lua as scripting language to program your game or application

I prefer using something like this in the first paragraph and let this slightly more technical paragraph here in the Features session, I'll push a commit and ask you to review again. Thanks again!